### ISSUE TITLE
Task4.2 - Generate Data for 2 stage addition 

### ISSUE THREAD

**Issue by YuhaoDuuu** [AguaClara/2_stage_coag_addition#13]
des later

**Comment by barbaraoramah** [comment]
Went to lab and flushed the system at 3:10 pm. the prior effluent turbidity was roughly at 17 NTU.

**Comment by ChingPangggg** [comment]
Manually flushed the system at 160 RPM for the water pump. The coagulant pump is at the next increment (56).

**Comment by YuhaoDuuu** [comment]
April 24 conducted 2 stage coag addition. coag pump speed started with 56 rpm. maximum coagulant dosage is 3mg/L. mannual flush is required for today's experiment check points on 11AM 1PM 3PM 5PM 7PM. humic acid concentration is 0.2mg/L.

**Comment by ChingPangggg** [comment]
Check at 7 PM, pump speed is at 44 RPM and no floc blanket is seen. Effluent turbidity is at around 62 NTU. Manual flush state is conducted and coag pump is manually set to 56 RPM.

**Comment by ChingPangggg** [comment]
April 25 
2 stage coagulant addition. the color of the HA and clay tank didnt look right on yesterday, the color is like a "coffee latte", but in the past we used the "milk tea". We thought that there was too much HA in the tank. so today before the refill we did the clean up first. 

experiment parameter didn't change compare with April 24. Started with coag pump speed 51.

**Comment by ChingPangggg** [comment]
April 25
5:35 PM check
No floc blanket is observed
Effluent turbidity is at 80 NTU approximately
Manual flush was conducted and current coag pump speed is at 56.

**Comment by barbaraoramah** [comment]
May 3rd
3:00 PM check
As we could not find the HA, only using clay in our stock tank.
The absence of HA contributes to low turbidity observed at approx. 9 NTU. There was a floc blanket.
Manual flush was conducted and coag pump 1 is at 22.8, coag pump 2 is at 30. Water pump is at 29.

**Comment by ChingPangggg** [comment]
May 7th
2-stage experiment was run, but the lowest effluent turbidity was at around 68 for a long period.
The set-up is as followed:
6.0 mL PAC/2L water
17.0 g Clay and 0.2 g HA /5L  water
coag pump 1: 15.9 RPM
coag pump 2:?

Therefore, we switched back to running 1-stage.

**Comment by ChingPangggg** [comment]
May 7th
2-stage experiment was run with influent maintained at 100 NTU. 
The set-up is as followed:
6.0 mL PAC/2L water
17.0 g Clay and 0.2 g HA /5L water
coag pump 1: Increment function
coag pump 2: 2804
water pump: 29

**Comment by barbaraoramah** [comment]
May 9th
2-Stage experiment with an influent of 100 NTU
Set up :
14 mL PAC/ 4 L DI water 
17.0 g Clay and 0.2 g HA /5L water
Coag pump 1: increment function
Coag pump 2: 28.4
water pump 29

In under 20 minutes, the effluent turbidity went from approximately 60 NTU to around 2 NTU. There is also a floc blanket present. The size of flocs generated were bigger than ever achieved throughout the semester. This is surprising as this is the lowest effluent we have achieved this semester. This may be due to the fact that we did a thorough clean of the system the day before allowing the apparatus to work more efficiently.
